index.html

```diff
@@ -5156,17 +5156,89 @@ <h3>Overview</h3>
         and generating a JSON-LD document in expanded form for all
         <a>RDF literals</a>, <a>IRIs</a>
         and <a>blank node identifiers</a>.
-        If the <a data-link-for="JsonLdOptions">useNativeTypes</a> flag is set to <code>true</code>,
-        <a>RDF literals</a> with a
-        <a>datatype IRI</a>
-        that equals <code>xsd:integer</code> or <code>xsd:double</code> are converted
-        to a <a>JSON numbers</a> and <a>RDF literals</a>
-        with a <a>datatype IRI</a>
-        that equals <code>xsd:boolean</code> are converted to <code>true</code> or
-        <code>false</code> based on their
-        <a>lexical form</a>
-        as described in
-        <a class="sectionRef" href="#data-round-tripping"></a>.
+
+        If the <a data-link-for="JsonLdOptions">useNativeTypes</a> flag is set
+        to <code>true</code>, <a>RDF literals</a> are converted to JSON values
+        based on their <a>datatype IRI</a> and <a>lexical form</a> as described
+        in <a class="sectionRef" href="#data-round-tripping"></a>.
+      </p>
+
+      <table class="simple">
```

Comment:
The table formatting needs some work; probably vertical alignment and borders to make it understandable.

Reply:
Yes, this is a rather draft version; I'd like it to look like `<table class="simple">` in YAML-LD spec but I haven't looked yet into what CSS markup is needed for that.

Reply:
…ReSpec has changed table formatting: https://github.com/speced/respec/issues/4667

A pity, I liked the previous blue style.

Reply:
![image](https://github.com/user-attachments/assets/b2b3bd39-1c75-437f-a636-29db09aca1d1)

Updated.

Reply:
Yes, that's what I did to obtain the formatting on the screenshot above.